Cargo.toml

```diff
@@ -5,6 +5,8 @@ authors = ["The Servo Project Developers"]
 description = "Geometry primitives"
 documentation = "https://docs.rs/euclid/"
 repository = "https://github.com/servo/euclid"
+keywords = ["matrix", "vector", "linearalgebra", "geometry"]
```

Comment:
I believe the proper keyword is [linear-algebra](https://crates.io/keywords/linear-algebra)